### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by wstaelens**
first test

### CODE REVIEW

File: documentation/index.md [hunk 1]
```diff
@@ -2,3 +2,11 @@
 layout: page
 title: Documentation
 ---
+
+- Getting Started
+    - [Introduction](./introduction)
+    - [Obtain and Build the Code](./build)
+    - [Nuget](./nuget)
+- Ab ovo usque ad mala
```
Comment (batzen) [inline documentation/index.md]: Would be nice if we could stick to english.

**Comment by wstaelens** [comment]
added some extra (learning first steps with Git)

File: documentation/introduction.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Introduction
+---
+
+luent Ribbon Control Suite is a library that implements Microsoft® Office Fluent™ user interface for the Windows Presentation Foundation (WPF).
```
Comment (batzen) [inline documentation/introduction.md]: "Fluent", not "luent" ;-)

File: documentation/introduction.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+### Ribbon control
+
+What is a ribbon? A ribbon is a control which consists of several tabs, grouped by functionality to represent the menu actions of a software programme.
+
+![Ribbon overview](http://download-codeplex.sec.s-msft.com/Download?ProjectName=fluent&DownloadId=102041)
```
Comment (batzen) [inline documentation/introduction.md]: Please don't reference resources from CodePlex.
If you need images you should add them to this repository.

**Comment by batzen** [comment]
No problem if you are still learning git. 
I am no git expert too. ;-)
Just take your time.

**Comment by wstaelens** [comment]
I can't upload images to github via the browser so I used this workaround:

"The workaround I use, is to base64 encode images, here is one tool that will do that for you (it even offers image optimization): http://beta.base64-image.de/

Then you can just click copy image and that will give you a long string of characters that can be used inside of an 's src="", or a markdown embedded image alt text."

http://stackoverflow.com/a/33261668/187650

**Comment by batzen** [comment]
You should get a git client then. I don't think it's a good idea to handle images that way.

**Comment by wstaelens** [comment]
better now?

**Comment by batzen** [comment]
A lot better ;-)

**Comment by batzen** [comment]
If you want to setup the site locally to be able to preview your changes, just send me a message on gitter and I will try to get you up and running.

**Pull request merged**